.github/workflows/rust.yml

```diff
@@ -20,8 +20,19 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Coding style
-      run: cargo fmt -- --check
+    - uses: actions/setup-python@v3
+    - uses: pre-commit/action@v3.0.1
+      with:
+        extra_args: cargo-clippy-fix --all-files
+    - uses: pre-commit/action@v3.0.1
+      with:
+        extra_args: cargo-clippy --all-files
+    - uses: pre-commit/action@v3.0.1
+      with:
+        extra_args: cargo-check --all-files
+    - uses: pre-commit/action@v3.0.1
+      with:
+        extra_args: cargo-fmt --all-files
```

Comment:
Can you not run `pre-commit` once with `--files virtme_ng_init/src/*.rs` as extra arguments?

    pre-commit run --files virtme_ng_init/src/*.rs

Reply:
Edit: I forgot we were already in the `virtme_ng_init` dir here, so I guess the arguments can be `--files */*`:

    pre-commit run --files */*

Reply:
Addressed in https://github.com/arighi/virtme-ng/pull/329